Source/WebCore/editing/cocoa/AttributedString.mm

```diff
@@ -76,13 +79,13 @@
     }, [] (const RetainPtr<NSDate>& value) -> RetainPtr<NSObject> {
         return value;
 #if PLATFORM(MAC)
-    }, [] (const RetainPtr<NSFont>& value) -> RetainPtr<NSObject> {
-        return value;
+    }, [] (const Ref<Font>& font) -> RetainPtr<NSObject> {
+        return toNSFont(font->getCTFont());
     }, [] (const RetainPtr<NSColor>& value) -> RetainPtr<NSObject> {
         return value;
 #else
-    }, [] (const RetainPtr<UIFont>& value) -> RetainPtr<NSObject> {
-        return value;
+    }, [] (const Ref<Font>& font) -> RetainPtr<NSObject> {
+        return (UIFont *)font->getCTFont();
```

Comment:
We should use `bridge_cast` here (and if we can't use `bridge_cast` right now, we should fix it so that we can use it here).